Source/CesiumRuntime/Public/CesiumSampleHeightResult.h

```diff
@@ -28,5 +28,5 @@ struct CESIUMRUNTIME_API FCesiumSampleHeightResult {
    * will have more information about the problem.
    */
   UPROPERTY(BlueprintReadWrite, Category = "Cesium")
-  bool SampleSuccess;
+  bool SampleSuccess{false};
```

Comment:
This is the biggest nitpick of all time, but in other classes or structs we tend to use `=` to initialize the default values, so I would like to change this to `bool SampleSuccess = false` 😅 Hopefully it's not too bad if I do that before merging